### PR TITLE
Share DFTX functions for create and update loan scheme

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind refactor

#### What this PR does / why we need it:
Create and update loan scheme should use setLoanScheme functions

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

**Review by canonbrother — DISMISSED** [review]

**Review by surangap — APPROVED** [review]

**Review by surangap — COMMENTED** [review]
@jingyi2811  this came into my mind later.
can we combine both test files to one? `packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/UpdateLoanScheme.test.ts` and `packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/CreateLoanScheme.test.ts` to `packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/SetLoanScheme.test.ts`

**Review by surangap — CHANGES_REQUESTED** [review]
https://github.com/DeFiCh/jellyfish/pull/827#pullrequestreview-807036884

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_set_loan_scheme.test.ts [hunk 1]
```diff
@@ -1,89 +1,249 @@
...
-  loanTokenId = await testing.container.call('setloantoken', [{
-    symbol: 'Token1',
-    name: 'Token1',
-    fixedIntervalPriceId: 'Token1/USD',
-    mintable: true,
-    interest: new BigNumber(0.01)
-  }, []])
-  await testing.generate(1)
-})
+describe('loan.createLoanScheme()', () => {
```
Comment (ivan-zynesis) [inline packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_set_loan_scheme.test.ts:18]: Edited wrong file? This file is `upload_loan_token` test, how come changed to `set_loan_scheme`?

**Review by ivan-zynesis — CHANGES_REQUESTED** [review]

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_set_loan_scheme.test.ts (hunk 1 above)
Comment (ivan-zynesis) [reply packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_set_loan_scheme.test.ts:18]: I read correctly right, need more eyes
image

Comment (jingyi2811) [reply packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_set_loan_scheme.test.ts:18]: Good catch thanks.

**Review by surangap — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**